<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaestroControlWidget</class>
 <widget class="QWidget" name="MaestroControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Maestro Control Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="sectionTab">
      <attribute name="title">
       <string>Section</string>
      </attribute>
      <layout class="QGridLayout" name="sectionLayout"/>
     </widget>
     <widget class="QWidget" name="animationTab">
      <attribute name="title">
       <string>Animation</string>
      </attribute>
      <layout class="QGridLayout" name="animationLayout"/>
     </widget>
     <widget class="QWidget" name="canvasTab">
      <attribute name="title">
       <string>Canvas</string>
      </attribute>
      <layout class="QGridLayout" name="canvasLayout"/>
     </widget>
     <widget class="QWidget" name="showTab">
      <attribute name="title">
       <string>Show</string>
      </attribute>
      <layout class="QGridLayout" name="showLayout"/>
     </widget>
     <widget class="QWidget" name="deviceTab">
      <attribute name="title">
       <string>Devices</string>
      </attribute>
      <layout class="QGridLayout" name="deviceLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
